feat(JS): Add `normalizeMaxBreadth` option (#4844)
getsentry/sentry-docs: src/platforms/common/configuration/options.mdx

<ConfigKey name="normalize-depth" supported={["javascript", "node"]}>

> Sentry SDKs normalize any contextual data to a given depth. Any data beyond this depth will be trimmed and marked using its type instead (`[Object]` or `[Array]`), without walking the tree any further. By default, walking is performed `3` levels deep.

Reviewer: i know this isn't your edit, but the `3` is a bit of a cheat. I've done an edit that maintains the formatting, but another way to deal with it is: "By default, walking is performed three levels deep." Pick your poison! =)

```suggestion
Sentry SDKs normalize any contextual data to a given depth. Any data beyond this depth will be trimmed and marked using its type instead (`[Object]` or `[Array]`), without walking the tree any further. By default, the walking depth level is set to `3`.
```

Author: Fair. In our copious collective free time, we actually probably should go through this whole page and standardize how we specify defaults, if nothing else. They're pretty inconsistent.

</ConfigKey>

> <ConfigKey name="normalize-max-depth" supported={["javascript", "node"]}>

Reviewer 2: ```suggestion
<ConfigKey name="normalize-max-breadth" supported={["javascript", "node"]}>

This is the maximum number of properties or entries which will be included in any given object or array when the SDK is normalizing contextual data. Any data beyond this depth will be dropped. (defaults to 1000)
```

I might be missing somethieng but the PR description referrs to `normalizeMaxBreadth` so could this be a typo here?

Author: 100% copy 🍝. Thanks for catching!

> <ConfigKey name="normalize-max-breadth" supported={["javascript", "node"]}>

Reviewer 3: Please set React Native as notSupported for now.

> 

Reviewer 4: ```suggestion

(Available in version 6.X.X and above)

```

> This is the maximum number of properties or entries which will be included in any given object or array when the SDK is normalizing contextual data. Any data beyond this depth will be dropped. (defaults to 1000)

Reviewer: ```suggestion
This is the maximum number of properties or entries that will be included in any given object or array when the SDK is normalizing contextual data. Any data beyond this depth will be dropped. (defaults to 1000)
```

Author: Ugh. This is literally the one grammar thing where the wrong way is what sounds more natural to my ears. I'll try to pay more attention.

Reviewer: You may have somehow picked up on a British English way of doing it. My Aussie tech writing friend informed me the other day that we do this differently. It's a bit more interchangeable in British English apparently.
